Src/PChecker/CheckerCore/SystematicTesting/Strategies/Feedback/Generator/Mutator/RandomInputMutator.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.IO;
+using PChecker.Generator.Object;
+
+namespace PChecker.Generator.Mutator;
+
+public class RandomInputMutator : IMutator<RandomInputGenerator>
+{
+    private readonly int _meanMutationCount = 10;
```

Comment:
Why are these values 10 and not something else?

Reply:
This is a hyper-parameter. We can make this configurable in the future.